src/tools/annotation/EllipticalRoiTool.test.js

```diff
@@ -16,11 +16,7 @@ jest.mock('./../../externalModules.js', () => ({
     metaData: {
       get: jest.fn(),
     },
-    getPixels: () => {
-      return [100, 100, 100,
-        100, 4, 5,
-        100, 3, 6];
-    }
+    getPixels: () => [100, 100, 100, 100, 4, 5, 100, 3, 6],
```

Comment:
Same here? Sometimes it's easier to visualize if the shape matches how you would imagine the canvas.